com.chartboost.mediation/Runtime/AdFormats/Banner/Unity/ChartboostMediationUnityBannerAd.Creator.cs

```diff
@@ -0,0 +1,101 @@
+using Chartboost.Banner;
+using Chartboost.Utilities;
+using UnityEditor;
+using UnityEngine;
+using static Chartboost.Utilities.Constants;
+
+namespace Chartboost.AdFormats.Banner.Unity
+{
+    public partial class ChartboostMediationUnityBannerAd : MonoBehaviour
```

Comment:
Does `ChartboostMediationUnityBannerAd` require an `Image` to work? If so look into [Require Component](https://docs.unity3d.com/ScriptReference/RequireComponent.html)

Reply:
It does not. I had an image earlier for debugging, it's removed now.